build.sbt

```diff
@@ -31,33 +31,39 @@ lazy val backend = (project in file("backend"))
   .dependsOn(services)
   .dependsOn(core % "test->test")
 
+lazy val sfsBackend = (project in backendRoot / "sfs")
```

Comment:
I'm going to need a suggestion. It is both a abstract supertype _and_ a usable backend. The config backend is in there. And AFAIK, we will not have an LSF backend as a artifact. I hope to remove the SGE backend artifact in a commit soon, and just have the memory values be fed into a WDL `instantiateCommand`.